sky/cli.py

```diff
@@ -1379,7 +1379,13 @@ def autostop(
 
 
 @cli.command(cls=_DocumentedCodeCommand)
-@click.argument('clusters', nargs=-1, required=True)
+@click.argument('clusters', nargs=-1, required=False)
```

Comment:
If we are making this an optional argument, we should add a manual to check to ensure _either_ `clusters` are specified or `-a` is specified. If both are not specified, we should raise an error like we had before:

Before this PR:
```
(base) romilb@romilbx1yoga:$ sky start
Usage: sky start [OPTIONS] CLUSTERS...
Try 'sky start -h' for help.

Error: Missing argument 'CLUSTERS...'.
```


After this PR, there's no output. We should add a similar warning
```
(base) romilb@romilbx1yoga:$ sky start
(base) romilb@romilbx1yoga:$
```